tests/format/js/tagged-template-expression-ts/non-null-optional-chaning.ts

```diff
@@ -0,0 +1,3 @@
+(a?.b!)``;
+
+(a?.b)!``;
```

Comment:
Can you add trailing new line to all the new tests?

Reply:
done!